installer/ClientInstaller.cs

```diff
@@ -11,13 +11,14 @@
 using System.Text.RegularExpressions;
 using System.Windows.Forms;
 using Microsoft.Deployment.WindowsInstaller;
+using System.Collections.Generic;
 using WixSharp;
 using WixSharp.CommonTasks;
 
 public class ClientInstaller : MumbleInstall {
 	public ClientInstaller(string version, string arch) {
 		string upgradeGuid = "D269FC55-4F2C-4285-9AA9-4D034AF305C4";
-		string[] binaries = null;
+		List<string> binaries;
```

Comment:
Why switch to a List in the first place? We do not modify it after initialization.

Reply:
I see that changes in the followup commit. :)